pt-br/starter/installing.md

```diff
@@ -48,24 +48,15 @@ e salve-o na lista de dependências. Por exemplo:
 
 <pre>
 <code class="language-sh" translate="no">
-$ npm install express --save
+$ npm install express
 </code>
 </pre>
 
 Para instalar o Express temporariamente não o inclua na lista
-de dependências, omita a opção `--save`:
+de dependências:
```

Comment:
I don't know if just cutting this off translates properly. Are you a native speaker / how did you check?

Reply:
![Screenshot 2021-12-03 at 6 20 51 PM](https://user-images.githubusercontent.com/58332678/144605523-3d53dc9c-dd44-44ff-bcf8-bbab92fac7a4.png)
😅

Reply:
Right, that is why I'm wondering, as it is not even good English once translated

Reply:
Should there be a ```,``` in between?